versions/3.0.md

```diff
@@ -2689,14 +2809,22 @@ example:
   "required": [
     "name"
   ],
-  "examples": [
-  {
-    "name": "Puma",
-    "id": 1
-  }, {
-    "name": "Ferguson",
-    "id": 2
-  }]
+  "examples": {
```

Comment:
The idea behind removing `examples` was that unlike other places, there's no `content` here and no media types described. Providing multiple examples for a model is out of context and doesn't serve a lot of value. You'd need to combine the schema with a media type to define how it's actually serialized. You could argue that even the `example` should be dropped for that reason.